CMakeLists.txt

```diff
@@ -399,6 +399,11 @@ else()
 endif()
 include(ThirdpartyToolchain)
 
+if (THRIFT_VERSION VERSION_LESS "0.11.0")
+  add_definitions(-DPARQUET_THRIFT_USE_BOOST)
+  message(STATUS "Using Boost in Thrift header")
+endif()
```

Comment:
Our toolchain build seems to be broken. If we wanted to be rigorous, we could pin Thrift 0.10.0 in the toolchain part of the build matrix

Reply:
Makes sense to add this option. Will include that.

Reply:
Ref: https://thrift.apache.org/lib/cpp